Dockerfile

```diff
@@ -22,6 +22,13 @@ RUN curl --fail https://sh.rustup.rs -sSf \
     rustup default "${RUST_VERSION}-x86_64-unknown-linux-gnu" && \
     rustup target add wasm32-unknown-unknown
 
+# Install yq. We use it to read YAML file(s).
+ARG YQ_VERSION=v4.34.1
+RUN wget \
```

Comment:
`curl -L` can be used instead of `wget` (then we don't need an additional dependency on `wget`)

Reply:
Done.